src/common/src/system_param/mod.rs

```diff
@@ -36,7 +36,7 @@ macro_rules! for_all_undeprecated_params {
         // Hack: match trailing fields to implement `for_all_params`
         $(, { $field:ident, $type:ty, $default:expr, $is_mutable:expr })*) => {
         $macro! {
-            { barrier_interval_ms, u32, Some(1000_u32), false },
+            { barrier_interval_ms, u32, Some(1000_u32), true },
```

Comment:
BTW, setting this value to an arbitrarily large number may cause OOM. Perhaps we should document a range of empirical values somewhere for reference.

Reply:
Added notice messages in the ALTER SYSTEM if barrier interval or checkpoint frequency is set too high.